src/main/kotlin/PSIDemo.kt

```diff
@@ -0,0 +1,8 @@
+class PSIDemo {
+    fun sayHello() {
+        // TODO: test-123
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tokgozmusa/detekt-danger/blob/21fc906cb70154b97a030890e073c42583fc4bb8/src/main/kotlin/PSIDemo.kt#L3"></span>This comment contains text that has been defined as forbidden in detekt.</td>
    </tr>
  </tbody>
</table>